Dockerfile

```diff
@@ -3,4 +3,4 @@ ADD build/libs/payments.jar /srv/payments.jar
 WORKDIR /srv
 EXPOSE 8080
 RUN mkdir config
```

Comment:
This is the folder that contains our mailtemplates. Is it possible to map this directory to somewhere outside the container to keep the mailtemplates between multiple deployments?